tooling/nargo_cli/src/cli/test_cmd.rs

```diff
@@ -53,6 +55,40 @@ pub(crate) struct TestCommand {
     /// Number of threads used for running tests in parallel
     #[clap(long, default_value_t = rayon::current_num_threads())]
     test_threads: usize,
+
+    /// Configure formatting of output
+    #[clap(long)]
+    format: Option<Format>,
+
+    /// Display one character per test instead of one line
+    #[clap(short = 'q', long = "quiet", action)]
+    quiet: bool,
+}
+
+#[derive(Debug, Copy, Clone, clap::ValueEnum)]
+enum Format {
+    /// Print verbose output
+    Pretty,
```

Comment:
Would `Verbose` itself be a more fitting name, as the opposite of "terse"? Not sure it's trying to be beautiful. 
OTOH might have `JsonPretty` at some point I suppose, whereas `JsonVerbose` is not a thing 🤷

Reply:
I copied this from Rust:

```
        --format pretty|terse|json|junit
                        Configure formatting of output:
                        pretty = Print verbose output;
                        terse = Display one character per test;
                        json = Output a json document;
                        junit = Output a JUnit document
```

It's interesting because even the description for "pretty" says "verbose" 🤣 

I'll change it to verbose. It'll be different from Rust, which is something we try to avoid, but it's unlikely someone will pass `--format verbose` because that's the default.

Reply:
No no leave it, let’s not surprise anyone who is familiar with it!

Reply:
Ah, okay. Right, I copied Rust for those familiar with Rust. I'll rever it then :-)